src/index.ts

```diff
@@ -59,12 +59,11 @@ app.on("ready", async () => {
   // coreGraphInterpreter.run();
 
   blix = new Blix();
-  const initMainApis = exposeMainApis(blix);
```

Comment:
I did not like this callback funciton this much, because I couldn't think a proper reason for initializing the Main APIs twice in a sense. Can't see how this would have resolved them from being called Blix is instantiated.